packages/strapi-design-system/package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "compute-scroll-into-view": "^1.0.17",
+    "match-sorter": "^6.3.0",
```

Comment:
Could you please make it a dev dependencies since it's only used in storybook? 🙏🏻